### DISCUSSION TITLE
Helm chart should provide log rotation

### DISCUSSION THREAD

**Discussion by malthe** [apache/airflow#25837] [General]
### Description

The Helm chart should provide log rotation functionality for the various services in order to periodically clean up logs.

### Use case/motivation

While log rotation for workers can be implemented using a DAG, it fits more naturally on the operating system level.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
It already does. https://airflow.apache.org/docs/helm-chart/stable/parameters-ref.html (look for logGroomer). Maybe it needs to be surfaced better (feel free to make PR with docs update @malthe) but I believe it's all there. It could have more features for sure, but I think what is there is quite enough ? 

Converting into discussion in case there is more to it that I see

**Reply by potiuk** [reply]
You can even specify your own command if you want to "groom" the logs in a more fancy way.

**Comment by malthe** [comment]
We'll give it a test drive and see if docs could use more information. I think just one first step could be to add "log rotation" to the documentation because I would have not thought about searching for "grooming".

**Reply by potiuk** [reply]
Quite agree :) . I think grooming is very rarely used in this context
